js/flightlog_fields_presenter.js

```diff
@@ -502,16 +512,26 @@ function FlightLogFieldPresenter() {
                 return (value + 1500).toFixed(0) + " us";
             case 'rcCommand[3]':
                 return value.toFixed(0) + " us";
-                
+
             case 'motor[0]':
-            case 'motor[1]':            
-            case 'motor[2]':            
-            case 'motor[3]':            
-            case 'motor[4]':            
-            case 'motor[5]':            
-            case 'motor[6]':            
-            case 'motor[7]':            
-                return Math.round(flightLog.rcMotorRawToPct(value)) + " %";
+            case 'motor[1]':
+            case 'motor[2]':
+            case 'motor[3]':
+            case 'motor[4]':
+            case 'motor[5]':
+            case 'motor[6]':
+            case 'motor[7]':
+                return `${flightLog.rcMotorRawToPctEffective(value).toFixed(1)} %`;
```

Comment:
This should still include the raw value for reference and the percentage should be marked as relative to avoid confusion, so probably `${flightLog.rcMotorRawToPctEffective(value).toFixed(1)} % relative (${value})`.

Reply:
In the old code it has not raw value, but I can add it too without problem. The same concern about the size but I will try.